package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "auth-i",
+  "version": "1.0.0",
+  "description": "* Authentication\r * Express Middleware\r * Mongoose Middleware and Methods\r * Sessions and Cookies",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js"
```

Comment:
Remember that if your `main` points to `server.js`, this `start` command can just be `"nodemon"`. If you want.